pkgs/development/libraries/libunwind/default.nix

```diff
@@ -9,6 +9,11 @@ stdenv.mkDerivation rec {
     sha256 = "1y0l08k6ak1mqbfj6accf9s5686kljwgsl4vcqpxzk5n74wpm6a3";
   };
 
+  # There's no "gcc_s" (shared gcc lib) on musl.
```

Comment:
According to this, libunwind is broken on musl:

https://github.com/libunwind/libunwind#libc-requirements

We should probably switch to using LLVM's libunwind. I've added it for llvm/8/, but it may make sense to add it here as well:

https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/compilers/llvm/8/libunwind.nix